pytket/extensions/qiskit/backends/aer.py

```diff
@@ -237,8 +237,14 @@ def process_circuits(
         circuits: Sequence[Circuit],
         n_shots: Union[None, int, Sequence[Optional[int]]] = None,
         valid_check: bool = True,
-        **kwargs: KwargTypes,
+        **kwargs: Union[bool, int, float, str, None],
     ) -> List[ResultHandle]:
+        """
+        See :py:meth:`pytket.backends.Backend.process_circuits`.
+        Supported kwargs: `seed`, `postprocess`, `seed_auto_increase`.
+        seed_auto_increase=True will automatically increase the seed by one for the
```

Comment:
Remove.

Reply:
Done in [85489ef](https://github.com/CQCL/pytket-qiskit/pull/197/commits/85489ef983300e4739976ba68048e55e5ceb932a)